git-squash-tags

```diff
@@ -8,16 +8,18 @@ if [ $# -eq 1 ]; then
 fi
 
 echo "Grabbing tags following pattern $tag_prefix..."
-tags=(`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk | sed 's/ (tag: //' | sed 's/)//'`)
+tags=(`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk |grep 'tag:' | sed 's/.*tag: //' | sed 's/)//'|sed 's/,.*//'`)
 
 # pull out first tag and remove it from $tags as it will be used as base
 tags_first=${tags[0]}
 tags=${tags[@]:1}
 echo "Using $tags_first as base."
+NEWLINE=$'\n'
```

Comment:
I followed one of solutions recommended on http://stackoverflow.com/questions/3005963/how-can-i-have-a-newline-in-a-string-in-sh . Maybe it is not the best one, I do not write bash scripts often.